gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/PartitionedRegionStats.java

```diff
@@ -1238,6 +1238,10 @@ public void incPRNumRowsInCachedBatches(int inc){
     this.stats.incLong(prNumRowsInCachedBatches, inc);
   }
 
+  public void setPRNumRowsInCachedBatches(int value){
+    this.stats.setLong(prNumRowsInCachedBatches, value);
```

Comment:
If the increment method is not required anymore shouldn't we remove it at all and just keep the setter method.